src/main/java/com/offdk/play/model/game/Player.java

```diff
@@ -0,0 +1,29 @@
+package com.offdk.play.model.game;
+
+import com.google.common.collect.Maps;
+import com.offdk.play.model.slack.User;
+import com.offdk.play.rating.RatingKey;
+import com.offdk.play.rating.RatingType;
+import java.math.BigDecimal;
+import java.util.Map;
+import java.util.Optional;
+
+public class Player {
+
+  private final User user;
+
+  private Map<RatingType, Map<RatingKey, BigDecimal>> currentRating;
```

Comment:
I would think so. In the data model, wouldnt you want to have the Player's rating associated with them? How else would you rank all the players?

Also, it's quite expensive to recalculate the ratings, mainly because order matters. If player A has a score of 1500 and player B has a score of 1500, maybe after their game it's now 1510/1490. After the game, Player C challenges Player B, in a 1500 v 1490 matchup, and C wins, which makes B's rating drop to like 1483. Now if you wanted to recalculate A's rating, you couldnt use B's current rating, you'd need to know their rating at the time of the match. And the only way to do that is either have it stored on the match, or recalculate every game, even those not involve A or B, just to get back to the point where you have enough information.